.github/workflows/pull_request_label.yml

```diff
@@ -0,0 +1,18 @@
+name: PR label
```

Comment:
Total nit, can this just be called "PR"? Or does the name need to be unique? I think there's enough context from the job name.

<img width="574" alt="Screenshot 2024-11-04 at 16 41 36" src="https://github.com/user-attachments/assets/9574c9d8-d324-49e2-b624-017a572bf4a0">

Reply:
No worries, no need for it to be unique, they can be uniquely referenced using the yaml file if needed. The listing now looks like this:
```
Stage  Job ID               Job name                      Workflow name  Workflow file           Events
0      unit-tests           Unit Tests                    Main           main.yml                push,schedule
0      integration-tests    Integration Tests             Main           main.yml                schedule,push
0      swift-license-check  Swift License Header Check    PR             pull_request.yml        pull_request
0      unit-tests           Unit Tests                    PR             pull_request.yml        pull_request
0      integration-tests    Integration Tests             PR             pull_request.yml        pull_request
0      cxx-interop          Cxx Interop                   PR             pull_request.yml        pull_request
0      soundness            Soundness                     PR             pull_request.yml        pull_request
0      semver-label-check   Semantic Version Label Check  PR             pull_request_label.yml  pull_request
```